<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
        <param name="rate" value="9"/>
    </node>
    <!-- <node name="gencam_cu" pkg="gencam_cu" type="gencam_cu">
        <param name="image_hz" value="50" />
    </node> -->
    <!-- launch-prefix="valgrind - -tool=callgrind" --> 
    <node name="line_processing" pkg="line_processing" type="line_processing" output="screen" launch-prefix="nice -n 19">
        <param name="birds_eye" value="/home/william/automow/automow/au_automow_vision/line_processing/birdeye_convert_mat.xml" />
        <param name="thresh_subtract" value="50" />
        <param name="percent_coverage" value="0.25" />
        <!-- params go here, we need to make some of the settings of the algorithm dynamic -->
    </node>
</launch>
